app/helpers/response_generation.py

```diff
@@ -0,0 +1,12 @@
+from flask import make_response, jsonify
+
+
+def make_error_msg(code, msg):
+    return make_response(jsonify({'success': False, 'error': {'code': code, 'message': msg}}), code)
+
+
+def create_response(base_url, short_id, error_message=None):
+    response = {"shorturl": base_url + short_id}
```

Comment:
The answer is missing the `'success': True` key see https://github.com/geoadmin/doc-guidelines/pull/43/files